app/assets/stylesheets/forever_style_guide/modules/_nav-icons.scss

```diff
@@ -9,71 +15,71 @@ $cart-count-width: 20px;
 }
 
 .nav-icon-stack {
-  width: $nav-icon-width;
-  padding: 7px 4px 4px 4px !important; // overriding .navbar-nav > li > a
+  padding: 3px 4px 4px 4px !important; // overriding .navbar-nav > li > a
   text-align: center;
 
   .fa {
     display: block;
     position: relative;
     top: $nav-icon-margin;
   }
-}
-.nav-icon-label {
-  font-size: $font-size-100;
-  line-height: $font-size-400;
-
-  &.has_count {
-    position: absolute;
-    top: 26px; // align labels
-  }
-}
 
-// Style offcanvas account icons consistent w/ product dropdowns
-@media (max-width: $grid-float-breakpoint-max) {
-  .navbar-account_item .nav-icon-stack {
-    width: auto;
-    padding: 15px !important; // override .navbar .nav.navbar-nav > li > a
-    text-align: left;
+  .icon-bar {
+    display: block;
+    width: 28px;
```

Comment:
magic numbers here that are the same as variables already defined. there's an opportunity to be a little less explicit w/ the variable names and share them throughout here.